packages/ui/src/components/PasswordStrengthMeter/index.tsx

```diff
@@ -76,6 +76,9 @@ const DEFAULT_ESTIMATE = () => ({ score: 0 as Score })
 const DEFAULT_FORBIDDEN_WORDS: PasswordStrengthMeterProps['forbiddenInputs'] =
   []
 
+/**
+ * @deprecated use Meter component instead
```

Comment:
You need to also mark it deprecated on storybook in file `packages/ui/src/components/PasswordStrengthMeter/__stories__/index.stories.tsx` with those parameters:

```ts
export default {
  component: PasswordStrengthMeter,
  parameters: {
      deprecated: true,
      deprecatedReason:
         'This component is deprecated, please use Meter instead.',
  },
  title: 'Components/Feedback/PasswordStrengthMeter',
} as Meta<typeof PasswordStrengthMeter>
```